autoprof/models/nonparametric_model.py

```diff
@@ -6,6 +6,7 @@
 from .ray_model import Ray_Galaxy
 from .wedge_model import Wedge_Galaxy
 from ._shared_methods import parametric_initialize
+
 import numpy as np
 import torch
```

Comment:
numpy and torch should be imported above the local imports

Reply:
I'll be solving each one soon 
Thank God the problem was something else
I thought my whole night's work went down the drain!